homeassistant/components/command_line/notify.py

```diff
@@ -44,8 +46,31 @@ def __init__(self, command: str, timeout: int) -> None:
 
     def send_message(self, message: str = "", **kwargs: Any) -> None:
         """Send a message to a command line."""
+        command = self.command
+        if " " not in command:
+            prog = command
+            args = None
+            args_compiled = None
+        else:
+            prog, args = command.split(" ", 1)
+            args_compiled = Template(args, self.hass)
```

Comment:
Why not treat the whole command as a template, instead of just the arguments?

Reply:
It was done for `sensor` and `binary_sensor` long before my time but I assume it's a "security" aspect to not allow changing the command but only the arguments after it has been created.

I think we should just repeat the same here for notify and allowing to change the command from the template should probably be discussed in a new PR in such case.